liblangutil/ParserBase.h

```diff
@@ -103,8 +103,12 @@ class ParserBase
 	void fatalParserError(ErrorId _error, std::string const& _description);
 	void fatalParserError(ErrorId _error, SourceLocation const& _location, std::string const& _description);
 
+	/// Creates a @ref ParserInfo and annotates it with the current position and the
+	/// given @a _description.
+	void parserInfo(ErrorId _error, SourceLocation const& _location, std::string const& _description);
```

Comment:
This isn't used by anything, is it needed?

Reply:
Ah true, it was used by the SPDX "info", forgot to remove this.